### DISCUSSION TITLE
page that opens dynamically

### DISCUSSION THREAD

**Discussion by LeChatSoeur** [awcodes/filament-curator#469] [Q&A]
Hi,

How to integrate the page directly into Filament? Being able to see the images in the library, the drag and drop, etc., without opening the dynamic page?

With <x-curator::modals.modal />?

https://github.com/awcodes/filament-curator/assets/66835174/da969ad8-4d4b-444d-a8fa-5be2761a58ae

**Comment by awcodes** [comment]
Sorry, I don't understand what you are asking.

**Reply by LeChatSoeur** [reply]
In the video, starting from 2 seconds, the window that appears, I want to integrate it directly, as if it's a Filament page. Like a standard form but in the form of a library and upload feature.

**Reply by awcodes** [reply]
Well, it's a filament form field. So it's just not going to work with filament unless it's a CuratorPicker form field in the form schema.

**Reply by LeChatSoeur** [reply]
Ok thanks
